tests/unit/helpers.js

```diff
@@ -346,12 +347,14 @@ export function dispatchClientMetadata({
   store = createStore().store,
   clientApp = CLIENT_APP_ANDROID,
   lang = 'en-US',
+  regionCode = 'CA',
```

Comment:
I feel like it should be `null` by default, we're not going to restrict frequently.